pkg/config/config_template.yaml

```diff
@@ -21,6 +21,19 @@ api_key:
 #
 # dd_url: https://app.datadoghq.com
 
+## @param cloud_provider - string - optional - default: <empty>
+## This option restrict which endpoint will be used by the agent to retrieve host metadata. By default the agent will try
+## all cloud provider.
+## Possible values are:
+## "AWS"     AWS EC2, ECS/Fargate
+## "GCP"     Google Cloud Provider
+## "Azure"   Azure
+## "Alibaba" Alibaba
+## "none"    does not query any cloud metadata endpoints (falling back on system metadata). Using this setting on a cloud provider host may result in duplicated hosts in your account.
+## ""        default: queries all cloud metadata endpoints
+#
+# cloud_provider: ""
+
```

Comment:
What do you think about using `"all"` for the default value as `""` could means none?

Reply:
I agree, I changed default to be all, which also resolves the unclear debug message above